### ISSUE TITLE
[Feature Request] add support for FabricTailor and SkinRestorer skins

### ISSUE THREAD

**Issue by elhertz** [QuiltServerTools/BlockBot#26]
IMO SkinRestorer is better as it doesn't force a cape on you when you choose a player skin, but FabricTailor has a nice GUI.
I don't know which one to pick for my server but I'll go for the one it's easier for you to add support to (if).
Hope it is possible, thanks!

**Comment by emielderckx** [comment]
Oh so you like piracy.... skinrestorer is made for offline mode servers. I see no reason to support that

**Comment by Tom-The-Geek** [comment]
FabricTailor's primary purpose is to allow changing of skins on regular server/in singleplayer, so I see no reason for it to be unsupported; however, SkinRestorer will probably never be supported officially, as it is specifically designed for offline mode servers.

Also, @emielderckx your phrasing sounds a little like you are attacking the issue author, there are better ways of explaining that piracy is unsupported :P

**Comment by elhertz** [comment]
@emielderckx you are right. I should have thought about it before asking for support. My bad.
@Tom-The-Geek thank you for taking my request in consideration.

**Comment by samolego** [comment]
Hi there, author of FabricTailor here :).
You could request skin value from FT's interface `TailoredPlayer`
(see [code](https://github.com/samolego/FabricTailor/blob/805cddf60bc927eec1fc72167689607ed0ac1d05/src/main/java/org/samo_lego/fabrictailor/casts/TailoredPlayer.java#L30)). Then decode that string (it's encoded using Base64 format), what you're looking for is `SKIN` -> `url` value.

**Comment by samolego** [comment]
![Screenshot_20210912-210926679 (1)](https://user-images.githubusercontent.com/34912839/133000047-cbf8bad6-06bf-462c-8ad6-ae96ad87c2f3.jpg)
posting screenshot here as well

**Comment by samolego** [comment]
Leaving https://github.com/crafatar/crafatar/issues/54 for reference.

**Comment by samolego** [comment]
There seems to be a way to do this using https://mc-heads.net/
(comment in crafatar issue)
